### PR TITLE
Add About Ann

### PR DESCRIPTION
**Pull request by Ann924**
## Status

- Done

## GitHub Issues addressed

- This PR closes the issue of an out-of-date About page (for onboarding).

## What I did

- Added the file ENG_Ann_Li.jpg and modified team_info.json to include my profile picture.

## Screenshots

- Before
image

- After
image

## Tests

- Ran locally and passed the pre-commit tests.

## Questions/Discussions/Notes

-

## Merging the PR

- Who should merge this PR?
  - [ ] I will merge it myself
  - [x] The last reviewer to approve can merge it
- Should the commit history be preserved in the base branch, or is it okay to combine all of them into a single commit ("squash-merge")?
  - [ ] preserve the history
  - [ ] squash-merge

### CODE REVIEW

**Review by alex-shen1 — APPROVED** [review]
Note that this PR adds autoformatting changes that are _technically_ outside the scope of the PR, which we generally ought to try to avoid. With that being said, I'm pretty sure that it's just the product of the precommit hook, so good job on installing that if you did! Gonna approve the PR, should be good to merge once the merge conflict is fixed.

**Review by n4mahajan — APPROVED** [review]

**Pull request merged**
